app/jobs/articles/analytics_updater_job.rb

```diff
@@ -0,0 +1,11 @@
+module Articles
+  class AnalyticsUpdaterJob < ApplicationJob
```

Comment:
If you take a look at existing jobs you'll notice they are all named with the action as their first word, for example "bust cache job". 

It'd be great if you could follow the pattern, so in this case: `Articles::UpdateAnalyticsJob`

```suggestion
  class UpdateAnalyticsJob < ApplicationJob
```

Reply:
I have updated class name accordingly.